docs/crio.conf.5.md

```diff
@@ -344,6 +344,7 @@ The "crio.runtime.runtimes" table defines a list of OCI compatible runtimes.  Th
   "io.kubernetes.cri-o.userns-mode" for configuring a user namespace for the pod.
   "io.kubernetes.cri-o.Devices" for configuring devices for the pod.
   "io.kubernetes.cri-o.ShmSize" for configuring the size of /dev/shm.
+  "io.kubernetes.cri-o.sysfs-mount-rw" to mount sysfs writable when set to "true" (only safe combined with userns-mode).
```

Comment:
Can we restrict that to only work with userns mode?

Reply:
ping @dgl